content/en/blog/2024-05-16-modelix-platform-release-24.1/index.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2024-05-16
+title: "Modelix Platform Release 24.1"
+---
+
+Version **`24.1`** of the modelix platform is out now!
+
+The versions bundled in this release are:
+
+```
+modelixCoreVersion = 7.4.0
+incrementalVersion = 0.2.0
+buildToolsVersion = 1.5.0
+mpsPluginsVersion = 0.7.0
+```
+
+If you want to know how to use this platform in your Gradle build, check out the corresponding [documentation](https://docs.modelix.org/modelix/23.2/platform/howto/usage-platform.html).
+
+
+- [Feature Highlights](#feature-highlights)
+  - [Support for newer MPS Versions](#support-for-newer-mps-versions)
+  - [Stabilization of Bulk Model Synchronization Plugin](#stabilization-of-bulk-model-synchronization-plugin)
+- [Development Insights](#development-insights)
+  - [MPS Sync Plugin (experimental)](#mps-sync-plugin-experimental)
+  - [Adapter for LionWeb Bulk API (experimental)](#adapter-for-lionweb-bulk-api-experimental)
+  - [Modelix LinkedIn Page](#modelix-linkedin-page)
+
+# Feature Highlights
+
+## Support for newer MPS Versions
+
+Modelix components are now compatible with MPS 2023.2 and below.
+To decrease the required maintenance effort, we decided to drop support for MPS 2020.3.
+While some use cases might still work with 2020.3, we highly recommend using a more recent MPS version.
+
+## Stabilization of Bulk Model Synchronization Plugin
+
+The [bulk-model-sync Gradle Plugin](https://docs.modelix.org/modelix/23.2/core/reference/component-bulk-model-sync-gradle.html) allows bulk synchronization of models between MPS and a [model-server](https://docs.modelix.org/modelix/23.2/core/reference/component-model-server.html).
+It is now considered stable, but it is still actively being worked with regards to performance.
+
+# Development Insights
+
+## MPS Sync Plugin (experimental)
+
+We are hard at work to create a new Kotlin-based MPS Sync Plugin, which will allow you to bind local MPS repositories to remote model-server repositories.
+You will be able to seamlessly exchange model data between MPS and the model-server.
+Changes performed on one side will be automatically synchronized to the other side.
```

Comment:
This text could require a sentence on how this approach differs from the bulk sync plugin.

Reply:
And to the legacy sync plugin. 

By the way, have we talked about the legacy sync plugin as a standalone library in any of the release notes recently?

Reply:
I don't think so @benedekh. Could you try and put the difference between legacy and new sync plugin into words? You're more familiar with it than I am.

Reply:
I'm not that much familiar with the legacy plugin. Let's postpone it until the official release of the new sync plugin.

Reply:
We've clarified this a bit.